.pre-commit-config.yaml

```diff
@@ -218,39 +218,26 @@ repos:
     - tests/
     pass_filenames: false
 
-- repo: local
```

Comment:
This does not take into account the deps that were managed externally like `pylint-pytest` (see `tox.ini`) and forces copy-paste of the dependencies in multiple locations. Like keeping the runtime/package metadata and the test deps in sync with this hook setup.
Besides, it makes it impossible to implement a reproducible env for this check w/o the dependency tree transitive constraints pinned which is otherwise possible with externally-managed deps in tox.

Is this a trade-off we're willing to make?

Reply:
What about just leaving everything in a test-requirements file for both linting and testing and pull that from precommit and tox....  It would be an all-in-one like it is today.

Reply:
(I suggest this not nothing if it's possible :))

Reply:
It is possible to source the requirements files in tox but not pre-commit. It would mean that this specific hook will be running in the venv that tox creates when it's `local` as pre-commit will not create a venv for this case. And we'll need to keep `manual` because it won't work in pre-commit.ci.
Basically, keep what I did originally.